### PR TITLE
[kibiter] Use Bitergia logo during the Kibiter splash screen and in the navbar theme

### PR DESCRIPTION
**Pull request by acs**

### CODE REVIEW

**Comment by canasdiaz** [comment]
It does not work due to the image used by the CSS directive `background-image: url("~ui/images/bitergia-analytics.svg");` can not be found.

```
~ ls src/ui/public/images 
grimoire-dashboard.svg  kibana.svg
```

This is the error I got:
```
    ERROR in ./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"/home/luis/repos/chaoss_grimiorelab-kibiter/optimize/.cache/4d1a81b1c2518d19b1b197227e09d3a90c91c0ac/less"}!./node_modules/css-loader?{"importLoaders":2}!./node_modules/postcss-loader/lib?{"config":{"path":"/home/luis/repos/chaoss_grimiorelab-kibiter/src/optimize/postcss.config.js"}}!./node_modules/less-loader/dist/cjs.js!./src/ui/public/chrome/directives/global_nav/global_nav.less
    Module not found: Error: Can't resolve 'ui/images/bitergia-analytics.svg' in '/home/luis/repos/chaoss_grimiorelab-kibiter/src/ui/public/chrome/directives/global_nav'
```

File: src/ui/public/chrome/directives/global_nav/global_nav.less [hunk 1]
```diff
@@ -33,7 +33,7 @@
     width: @global-nav-open-width;
     list-style-type: none;
     &.kibana {
-      background-image: url("~ui/images/grimoire-dashboard.svg");
+      background-image: url("~ui/images/bitergia-analytics.svg");
```
Comment (canasdiaz) [inline src/ui/public/chrome/directives/global_nav/global_nav.less:36]: This image file must be submitted. :thinking:

**Review by canasdiaz — CHANGES_REQUESTED** [review]

File: src/ui/public/chrome/directives/global_nav/global_nav.less (hunk 1 above)
Comment (acs) [reply src/ui/public/chrome/directives/global_nav/global_nav.less:36]: Yes, my fault, adding the image ... done! @sanacl could you recheck? Thanks!

**Comment by canasdiaz** [comment]
The login when the content is being loaded is nice. The logo on the menu bar is not correct as it does not contain the words "Bitergia Analytics". @acs I can help with the image, give me one hour so I can provide it. Does it make sense?

**Comment by acs** [comment]
Sure, great! The logo that appears during the content loading is our loved Bitergia owl in SVG format, and with CSS, it is adjusted and a circle is added around it. And also, the fading effect is added. It is really nice. Once you have the correct logo, I can modify this PR to add it!

**Comment by canasdiaz** [comment]
@acs find attached my proposed changes
```
background-position: 30px 5px;
background-size: 110px 77px;
```
Find attached the image. The appear I've worked on is the following (not sure if you guys like it)
screenhost

[bitergia-analytics.svg.zip](https://github.com/chaoss/grimoirelab-kibiter/files/2136908/bitergia-analytics.svg.zip)

**Comment by canasdiaz** [comment]
@acs in Inkscape the style of the second line was bold but the result was the same. When exported it became bold. This has been fixed.

[bitergia-analytics.svg.zip](https://github.com/chaoss/grimoirelab-kibiter/files/2137075/bitergia-analytics.svg.zip)

screenhost

**Comment by acs** [comment]
@sanacl all issues fixed now and your work included, so I am merging this PR now in order to build the new kibiter docker image. Thank you for your review and work.

**Pull request merged**